packages/use-intl/src/react/useTimeZone.tsx

```diff
@@ -1,5 +1,6 @@
+import { TimeZone } from '../core/utils/TimeZones';
 import useIntlContext from './useIntlContext';
 
-export default function useTimeZone() {
+export default function useTimeZone(): TimeZone {
```

Comment:
The return type is wrong, also `undefined` is allowed. Better just let TypeScript infer the return type and don't annotate it additionally!